### ISSUE TITLE
Block Change and Decay do not prevent ice melting

### ISSUE THREAD

**Issue by Tencao** [FoxDenStudio/FoxGuard#69]
This is an extremely annoying bug when trying to design structures with lighting, I would prefer a ice melt option or tied into decay but currently nothing seems to be preventing ice from melting. To clarify as well, the block change is in the passive group

**Comment by gravityfox** [comment]
This issue is already known and a solution is being worked on.

Basically there's no special handling for ice at the moment.

**Comment by worm424** [comment]
Please do not put [bug] on issues

**Comment by Tencao** [comment]
But, it is a bug? Block Change is also suppose to cover Ice, and indirectly this could follow with other blocks that can change

**Comment by worm424** [comment]
Yes I understand but the fact your reporting it here it's assumed it is a bug

**Comment by worm424** [comment]
Yes I understand but the fact your reporting it here it's assumed it is a bug
Thanks for reporting

**Comment by gravityfox** [comment]
if it makes you feel better, i added a tag.

**Comment by Tencao** [comment]
It's not really about making me feel better, I just like informing the devs exactly what my ticket is, since sometimes I also post suggestions using the [suggestion] tag

**Comment by gravityfox** [comment]
See the thing is... 

I'm a developer, and a college student.

I have the competence to tell the difference by actually reading what was written. If I can't tell whether it's supposed to be a bug report or a suggestion, I'd let you know. Although when that happens i'm often not inclined to ask, as it is usually because the issue is in engrizh or because it is practically void of useful information.

Basically don't worry about tagging issues. I'll tag them if I need them.

**Comment by Tencao** [comment]
No worries, different developers have different preferences so I don't really mind

**Comment by szmarczak** [comment]
Hello. Any updates on this?

**Comment by ghost** [comment]
Hello!

Please be aware that many of the open issues may be closed soon, in preparation of the beta release of FoxGuard 2.0. We have not yet determined if we are going to create a new repo, or use the current one, however, if we use the current one, this issue will be closed.

If the issue occurs again at a later date in FoxGuard 2.0, please make a new issue and reference this one.

Thank you!
Joshua
